### PR TITLE
[12.0][IMP] account_banking_mandate: Show acc number end in name_get

### PR DESCRIPTION
**Pull request by carlosdauden**
@Tecnativa

### CODE REVIEW

File: account_banking_mandate/models/account_banking_mandate.py [hunk 1]
```diff
@@ -79,6 +79,16 @@ def _get_default_partner_bank_id_domain(self):
         'unique(unique_mandate_reference, company_id)',
         'A Mandate with the same reference already exists for this company!')]
 
+    def name_get(self):
+        result = []
+        for mandate in self:
+            name = mandate.unique_mandate_reference
+            acc_number = mandate.partner_bank_id.acc_number
+            if acc_number:
+                name = '{} [{}]'.format(name, acc_number[-4:])
```
Comment (pedrobaeza) [inline account_banking_mandate/models/account_banking_mandate.py]:
```suggestion
                name = '{} [...{}]'.format(name, acc_number[-4:])
```

Comment (carlosdauden) [reply account_banking_mandate/models/account_banking_mandate.py]: Done

**Review by pedrobaeza — APPROVED** [review]

**Review by sergio-teruel — APPROVED** [review]

**Comment by carlosdauden** [comment]
@alexis-via can you check this?

**Comment by carlosdauden** [comment]
Should we unify criteria to show a meaningful reference?:
https://github.com/OCA/bank-payment/blob/10b68fd5dfbfa0ccc59d6cd95fb92ba007273ad7/account_banking_sepa_direct_debit/models/account_banking_mandate.py#L17
https://github.com/OCA/bank-payment/blob/10b68fd5dfbfa0ccc59d6cd95fb92ba007273ad7/account_banking_sepa_direct_debit/models/account_banking_mandate.py#L51-L59

**Comment by pedrobaeza** [comment]
Yes, good idea!

**Comment by pedrobaeza** [comment]
Waiting for the unification you talked for merging this.

**Review by misern2 — APPROVED** [review]

**Pull request merged**